strangler-infusereo/code/ruby-service.rb

```diff
@@ -0,0 +1,37 @@
+require 'sinatra'
+require 'json'
+require 'net/http'
+
+set :port, 9494
+
+$users=[{name: 'foo', id: 1}]
```

Comment:
Thankfully we can sidestep this whole mess since we're not actually updating the hash.  It works just fine by switching to class vars.  Super nice having the integration tests already written to prove it, too!